modules/core/src/main/resources/scalafix-migrations.conf

```diff
@@ -142,6 +142,12 @@ migrations = [
     newVersion: "3.0.0",
     rewriteRules: ["github:typelevel/cats-effect/v3_0_0?sha=series/3.x"],
     scalacOptions: ["-P:semanticdb:synthetics:on"]
+  },
+    {
+    groupId: "ch.epfl.scala",
+    artifactIds: ["scalafix-testkit"],
+    newVersion: "0.9.28",
+    rewriteRules: ["github:scalacenter/scalafix.g8/v0_9_28"]
```

Comment:
I don't think the current layout will work, we should rename back to `scalafix` to honor https://scalacenter.github.io/scalafix/docs/developers/sharing-rules.html#using-github

Reply:
https://github.com/scalacenter/scalafix.g8/pull/21

Reply:
To be safe, should this be pinned to a specific commit (since there are no tags over there) ?

Reply:
ok

Reply:
something like this : `  rewriteRules: ["github:scalacenter/scalafix.g8/v0_9_28?sha=dba15e3"]`